assets/js/cart.js

```diff
@@ -1,153 +1,5 @@
 var storefrontApp = angular.module('storefrontApp');
 
-storefrontApp.controller('cartController', ['$rootScope', '$scope', '$timeout', 'cartService', 'catalogService', 'availabilityService', 'loadingIndicatorService', function ($rootScope, $scope, $timeout, cartService, catalogService, availabilityService, loader) {
```

Comment:
This code is dead, checkout controller is used instead and it confused me some times, so I decided to remove at now.